service/prometheus/merge.go

```diff
@@ -1,30 +0,0 @@
-package prometheus
```

Comment:
`merge.go` moved to `update.go`, so the diff is gross. I apologise. It essentially did completely change anyway.